src/modules/hotkeys/i18n/cs_CZ.root.json

```diff
@@ -74,14 +74,30 @@
             "title": "Hlavní okno"
         },
         "mission": {
+            "alliance": {
+                "focus": "Focus alliance response window",
+                "title": "Alliance",
+                "toggle": "Toggle 'post in alliance chat'"
+            },
+            "arr": {
+                "next": "next tab",
+                "previous": "previous tab",
+                "title": "Arr"
```

Comment:
I would prefer an all caps: `ARR`.
Same for all other translations except de_DE

Reply:
Changed in [d7a8d53](https://github.com/LSS-Manager/LSSM-V.4/pull/1717/commits/d7a8d53e6c12024ab560fb108f86293bf001a332)